### PR TITLE
Windows export rcedit improvements

### PR DESCRIPTION
**Pull request by Joseph-DiGiovanni**
- Allows rcedit to be executed via system PATH without specifying an executable location in editor settings.

- When rcedit is executed through wine on non-windows systems, disable unnecessary wine debug output and the wine-mono install message on first run.

- Minor code refactoring for improved readability.

### CODE REVIEW

File: platform/windows/export/export_plugin.cpp [hunk 1]
```diff
@@ -513,9 +497,24 @@ Error EditorExportPlatformWindows::_rcedit_add_data(const Ref<EditorExportPreset
...
+		}
+
+		if (wine_path.is_empty()) {
+			wine_path = "wine"; // try to run wine from PATH
+		}
+
+		args.push_front(rcedit_path);
+		rcedit_path = wine_path;
+		setenv("WINEDEBUG", "-all", true); // Disable wine debug messages
+		setenv("WINEDLLOVERRIDES", "mscoree=", true); // Prevent wine-mono install prompt on first run
```
Comment (Calinou) [inline platform/windows/export/export_plugin.cpp:516]: I suggest using `OS::get_singleton()->set_environment()` instead for consistency with the rest of the engine, even if this block is UNIX-specific.

Comment (Joseph-DiGiovanni) [reply platform/windows/export/export_plugin.cpp]: Just changed it, thanks.

**Comment by Joseph-DiGiovanni** [comment]
Sorry, accidentally closed the PR renaming the branch. Reopening.

**Comment by Joseph-DiGiovanni** [comment]
> Are you trying to make this PR fetch from another branch?

I was 😅 I saw there is another PR attempting to remove rcedit altogether in favor of a built in solution, so I'm not even sure it's worth resubmitting anyway.

**Pull request closed without merging**